.github/workflows/build-seissol.yml

```diff
@@ -0,0 +1,134 @@
+name: build-seissol
+on:
+  - push
+
+jobs:
+  dependencies:
+    name: dependencies
+    runs-on: ubuntu-22.04
```

Comment:
Please update to latest LTS 24.04 (fresh out of the box :D)

Reply:
Ubuntu 24.04 doesn't seem to work yet—it can't find a runner at least